_posts/2017-10-20-jbhc17-from-my-perspective.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "#jbhc17 from my perspective"
+date: 2017-10-20 14:03:00 +0200
+categories: meetup
+author_name: Jacek Smolak
+author_url : /author/jaceksmolak
+author_avatar: jaceksmolak
+show_avatar : true
+read_time : 4
+show_related_posts: false
+square_related:
+feature_image: posts/2017-10-22-jbhc17-from-my-perspective/poster.jpg
+---
+
+When I first heard about [JB][jbrains] coming to HolidayCheck to work with us I was excited.
+I haven't heard much about him then, so I began googling him. I did remember the [7-26-talk]
+(which is awesome, you need to check it out), but I didn't know much more.
+Once I got to learn about him, I already knew - I can't miss this opportunity. And I didn't.
+
+## The trip
+
+As I am from Poland, I (actually me and my colleagues), had to hop on plane and get to Munich. Once there, we
+went to the hotel, went out for a burger and a beer, got back, went to sleep. Morning came.
+We arrived to meeting place around 9. JB was already there, preparing.
+I approached him, shook his hand, exchanged few words. Seems like a nice guy - I thought.
+
+## First day
+
+I sat down in 2nd row, so that I would not miss a thing. Keynote began. I'll be frank. From the moment he started,
+till the very end I listened to everything he had to say. Imagine his 7-26-talk, but for few hours (with breaks of course ;)).
+It was all very interesting. JB has this skill to attract peoples attention and talk in a very interesting way.
+Imagine a lesson you liked in school, because anything you heard there was valuable and attracted your attention.
+It was the same.
+
+The main reason I chose to go to [#jbhc17][jbhc17] was TDD and craftsmanship concerning our daily job.
+But the majority of talks during the first day were not about that. I have to say, it got me worried a bit.
+It was interesting, but still. Little did I know that JB had a plan.
+He introduced a way of working in order to solve one problem: bottlenecks.
+And he did a great job. We learned how to find what slows us down and how to tackle this.
+He showed us how to work efficiently and what eventually matters - our clients, time to bringing features to them
+is the most important thing. The shorter the better, as that is all that matters.
+
+How?
+
+He started off with [Winston W. Royce's waterfall][waterfall] model, then, step by step,
+changed it into test first approach.
+From queueing theory, test first programming to TDD and BDD. He spoke a lot about throughput of our work,
+from raw material to final product. How it flows through the system, where the constraints lie.
+JB described how value stream mapping works, which is the technique used to discovering bottlenecks.
+You think you have a high efficiency in your team? 30-50%? Think again :)
+We did the exercise and it was ~6%. Not kidding.
+
+First day introduced a lot of very useful knowledge, and went by fast.
```

Comment:
*The* first day

Reply:
I am not so sure about this one.

Reply:
Pardon? English uses articles right?

“Cat meowed as she fell down the stairs”
vs
“The cat meowed as she fell down the stairs”

I don’t know what else to say. You need to use articles.. *shrug* ;)

Reply:
OK :)

Reply:
Definite articles can only be omitted in [specific exceptional cases](https://www.grammarly.com/blog/articles/), such as when referring to languages or sports.

I'm with @gilligan that _Day_ isn't one of them.

Reply:
@timoreimann actually I think its not about `Day` but rather about the word "First", which always  requires definite article. So combo `The first *something*` without exceptions should be always written with `the`

Reply:
@HeeL in my opinion it doesn't make any difference as it's still a _specific_ day we're referring to, regardless of a the adjective ordinal. A more precise explanation might be [here](https://english.stackexchange.com/questions/1587/using-the-before-ordinal-numbers).

Regardless, we seem to agree that the article is in order, so let's close this case. :-)